ui/panels-plugin/src/plugins/markdown/MarkdownPanel.tsx

```diff
@@ -12,69 +12,82 @@
 // limitations under the License.
 
 import { useMemo } from 'react';
-import * as DOMPurify from 'dompurify';
-import { marked } from 'marked';
-import { Box, Theme } from '@mui/material';
+import ReactMarkdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+import rehypeRaw from 'rehype-raw';
+import rehypeSanitize from 'rehype-sanitize';
+import { Box, Link, Paper, Table, TableHead, TableBody, TableRow, TableCell } from '@mui/material';
 import { PanelProps } from '@perses-dev/plugin-system';
 import { MarkdownPanelOptions } from './markdown-panel-model';
 
 export type MarkdownPanelProps = PanelProps<MarkdownPanelOptions>;
 
-function createMarkdownPanelStyles(theme: Theme) {
+function createMarkdownPanelStyles() {
   return {
     // Make the content scrollable
     height: '100%',
     overflowY: 'auto',
-    // Ignore first margin
+    // Ignore initial margin
     '& :first-child': {
       marginTop: 0,
     },
-    // Styles for <code>
+    // <code>
     '& code': { fontSize: '0.85em' },
     '& :not(pre) code': {
       padding: '0.2em 0.4em',
-      backgroundColor: theme.palette.grey[100],
+      backgroundColor: '#f5f5f5',
       borderRadius: '4px',
     },
     '& pre': {
       padding: '1.2em',
-      backgroundColor: theme.palette.grey[100],
+      backgroundColor: '#f5f5f5',
```

Comment:
Hard-code the background-color to a nice grey.

Reply:
does this look ok in dark mode too?

Reply:
Foiled again by dark mode! 

Just checked, it doesn't look OK in dark mode... which I think is part of why I used `theme.palette` to begin with... but the lightest grey (i.e. grey[100]) is a bit too dark in regular mode...

What a chicken or egg problem